app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -172,7 +173,8 @@ class HomeFragment : Fragment() {
     private val sessionControlInteractor: SessionControlInteractor
         get() = _sessionControlInteractor!!
 
-    private var sessionControlView: SessionControlView? = null
+    @VisibleForTesting(otherwise = Modifier.PRIVATE)
```

Comment:
nitpick (blocking): this is the wrong import. You should use `VisibleForTesting.PRIVATE`